test/plots/areaPlotTests.ts

```diff
@@ -2,162 +2,181 @@
+        let twoPointData = [{x: 0, y: 0}, {x: 1, y: 1}];
+        let simpleDataset = new Plottable.Dataset(twoPointData);
+        areaPlot = new Plottable.Plots.Area<number>();
+        areaPlot.addDataset(simpleDataset);
+        areaPlot.x((d: any) => d.x, xScale)
+                .y((d: any) => d.y, yScale);
+        areaPlot.y0(() => 0)
+                .attr("fill", () => fill)
+                .attr("stroke", (d, i) => d3.rgb(d.x, d.y, i).toString())
+                .renderTo(svg);
+      });
 
-    it("draws area and line correctly", () => {
-      let areaPath = renderArea.select(".area");
-      assert.strictEqual(TestMethods.normalizePath(areaPath.attr("d")), "M0,500L500,0L500,500L0,500Z", "area d was set correctly");
-      assert.strictEqual(areaPath.attr("fill"), "steelblue", "area fill was set correctly");
-      let areaComputedStyle = window.getComputedStyle(<Element> areaPath.node());
-      assert.strictEqual(areaComputedStyle.stroke, "none", "area stroke renders as \"none\"");
-
-      let linePath = renderArea.select(".line");
-      assert.strictEqual(TestMethods.normalizePath(linePath.attr("d")), "M0,500L500,0", "line d was set correctly");
-      assert.strictEqual(linePath.attr("stroke"), "#000000", "line stroke was set correctly");
-      let lineComputedStyle = window.getComputedStyle(<Element> linePath.node());
-      assert.strictEqual(lineComputedStyle.fill, "none", "line fill renders as \"none\"");
-      svg.remove();
-    });
+      it("draws area and line correctly", () => {
+        let content = areaPlot.content();
+        let areaPath = content.select(".area");
+        areaPlot.datasets().forEach((dataset: Plottable.Dataset) => {
+          let data = dataset.data();
+          let pointsInArea = getPointsInArea(areaPlot, data);
+          TestMethods.assertLinePathEqualToDataPoints(areaPath.attr("d"), pointsInArea, xScale, yScale);
+          assert.strictEqual(areaPath.attr("fill"), fill, "area fill was set correctly");
+          assert.strictEqual(areaPath.style("stroke"), "none", "area stroke renders as \"none\"");
+
+          let linePath = content.select(".line");
+          TestMethods.assertLinePathEqualToDataPoints(linePath.attr("d"), data, xScale, yScale);
+          assert.strictEqual(linePath.attr("stroke"), "#000000", "line stroke was set correctly");
+          assert.strictEqual(linePath.style("fill"), "none", "line fill renders as \"none\"");
+        });
+        svg.remove();
+      });
 
-    it("area fill works for non-zero floor values appropriately, e.g. half the height of the line", () => {
-      areaPlot.y0((d) => d.bar / 2);
-      areaPlot.renderTo(svg);
-      renderArea = (<any> areaPlot)._renderArea;
-      let areaPath = renderArea.select(".area");
-      assert.strictEqual(TestMethods.normalizePath(areaPath.attr("d")), "M0,500L500,0L500,250L0,500Z");
-      svg.remove();
-    });
+      it("fills for non-zero floor values appropriately", () => {
```

Comment:
Let's use a better test name here.  It seems to be something along the lines that we are checking the `y0` points?